### PR TITLE
Update temporary package info instead of saving when prefering or unprefering an attribution on Audit View

### PR DESCRIPTION
**Pull request by leslielazzarino**
Some tests are still missing

### Summary of changes

Update temporary package info instead of saving when prefering or unprefering an attribution on Audit View. Also the global saving button is disabled if an attribution has been changed to preferred.

### Context and reason for change

That prevents collisions with unsaved changes in the attribution.

### How can the changes be tested

Set-unset the preferred attribute.

### CODE REVIEW

File: src/Frontend/Components/ResourceDetailsAttributionColumn/ResourceDetailsAttributionColumn.tsx [hunk 1]
```diff
@@ -116,6 +119,17 @@ export function ResourceDetailsAttributionColumn(
...
+      dispatch(
+        unlinkAttributionAndSavePackageInfoIfSavingIsNotDisabled(
+          selectedResourceId,
+          attributionIdOfSelectedPackageInManualPanel as string, // showSaveGloballyButton is true, so attributionIdOfSelectedPackageInManualPanel is not null
+          temporaryDisplayPackageInfo,
+        ),
+      );
+      return;
+    }
+
```
Comment (MarkusObendrauf) [inline src/Frontend/Components/ResourceDetailsAttributionColumn/ResourceDetailsAttributionColumn.tsx:132]: i dont understand this change. why is it needed now, and not before?

**Review by MarkusObendrauf — CHANGES_REQUESTED** [review]

File: src/Frontend/state/actions/resource-actions/__tests__/all-views-simple-actions.test.ts [hunk 2]
```diff
@@ -273,11 +275,29 @@ describe('The load and navigation simple actions', () => {
     );
   });
 
-  it('sets and gets isSavingDisabled', () => {
+  // replace following 3 tests with a single parametric one
```
[the remark this answers is not in the document]
Comment (leslielazzarino) [reply src/Frontend/state/actions/resource-actions/__tests__/all-views-simple-actions.test.ts:278]: I think in this case it displays nicely what is expected when in an easy-to-read table

File: src/Frontend/Components/ResourceDetailsAttributionColumn/ResourceDetailsAttributionColumn.tsx (hunk 1 above)
Comment (leslielazzarino) [reply src/Frontend/Components/ResourceDetailsAttributionColumn/ResourceDetailsAttributionColumn.tsx:132]: Before ctrl+S was always triggering a global save. Now it triggers a local save, if the global save is disabled.

File: src/Frontend/state/actions/resource-actions/save-actions.ts [hunk 3]
```diff
@@ -57,8 +61,8 @@ import { openPopup } from '../view-actions/view-actions';
 import { getMultiSelectSelectedAttributionIds } from '../../selectors/attribution-view-resource-selectors';
 import { setMultiSelectSelectedAttributionIds } from './attribution-view-simple-actions';
 
-export function setIsSavingDisabled(
-  isSavingDisabled: boolean,
+export function setAllowedSaveOperations(
+  isSavingDisabled: AllowedSaveOperations,
```
Comment (MarkusObendrauf) [inline src/Frontend/state/actions/resource-actions/save-actions.ts:65]: rename this variable

File: src/Frontend/state/actions/resource-actions/types.ts [hunk 4]
```diff
@@ -221,7 +222,7 @@ export interface SetTargetSelectedAttributionIdAction {
 
 export interface SetIsSavingDisabled {
   type: typeof ACTION_SET_IS_SAVING_DISABLED;
```
Comment (MarkusObendrauf) [inline src/Frontend/state/actions/resource-actions/types.ts:224]: rename function and action name

**Review by MarkusObendrauf — CHANGES_REQUESTED** [review]
i think the refactoring tools missed some things

File: src/Frontend/state/actions/resource-actions/types.ts [hunk 5]
```diff
@@ -39,7 +40,8 @@ export const ACTION_SET_SELECTED_RESOURCE_ID =
 export const ACTION_SET_EXPANDED_IDS = 'ACTION_SET_EXPANDED_IDS';
 export const ACTION_SET_TARGET_SELECTED_RESOURCE_ID =
   'ACTION_SET_TARGET_SELECTED_RESOURCE_ID';
-export const ACTION_SET_IS_SAVING_DISABLED = 'ACTION_SET_IS_SAVING_DISABLED';
+export const ACTION_SET_ALLOWED_SAVE_OPERATIONS =
+  'ACTION_SET_IS_SAVING_DISABLED';
```
Comment (MarkusObendrauf) [inline src/Frontend/state/actions/resource-actions/types.ts:44]: this still needs to be changed

**Pull request merged**